src/Provider/Doctrine/Auditing/Transaction/TransactionProcessor.php

```diff
@@ -217,7 +217,7 @@ private function audit(array $data): void
             'object_id' => (string) $data['id'],
             'discriminator' => $data['discriminator'],
             'transaction_hash' => (string) $data['transaction_hash'],
-            'diffs' => json_encode($data['diff'], JSON_THROW_ON_ERROR),
+            'diffs' => json_encode(mb_convert_encoding($data['diff'], "UTF-8", "UTF-8"), JSON_THROW_ON_ERROR),
```

Comment:
Surprisingly CI passed here whereas it failed on the other PR targeting `2.x` branch. By the way I prefer single quotes for strings when possible, could you please change this?